myscripts.js

```diff
@@ -1,26 +1,77 @@
+// SELECTORS
+
 const collection = [];
+const addbtn = document.getElementById('add');
+// title = document.getElementById('title');
+const bookContainer = document.getElementById('bookContainer');
+let storageObject;
+
+const booksHtmlTemplate = (obj) => `<div class='bookCont'>
+  <label class='title'>${obj.title}</label><br>
+  <label id='author'>${obj.author}</label><br>
+  <button class='remove' type='button' onclick= value=''>Remove</button>
+  <hr>
+  </div>`;
+
+const createBook = (arr) => {
+  arr.forEach((element) => {
+    const dinContent = document.createDocumentFragment();
+    const newDiv = document.createElement('div');
+    newDiv.innerHTML = booksHtmlTemplate(element);
+    dinContent.appendChild(newDiv);
+    bookContainer.appendChild(dinContent);
+  });
+};
+
+const update = () => {
+  const elements = document.getElementsByClassName('remove');
+  const bookCont = document.querySelectorAll('.bookCont');
 
-function Book(title,author) {
-    this.title = title;
-    this.author = author;
-    
+  for (let i = 0; i < elements.length; i += 1) {
+    elements[i].addEventListener('click', () => {
+      // console.log('remove clicked');
+      if (bookCont[i].parentNode) {
+        bookCont[i].parentNode.removeChild(bookCont[i]);
+      }
+      collection.splice(i, 1);
+      localStorage.setItem('books', JSON.stringify(collection));
+    });
```

Comment:
- [ ] I think this block of code is buggy. When I remove a book from the list in the UI, it removes all the book objects from the local storage. Please carefully hunt the bug and fix the issue. :pray: 

![Screenshot from 2021-11-24 02-00-54](https://user-images.githubusercontent.com/64894927/143142881-a9f5ee48-21e6-494a-ae55-56cd9b1163f2.png)

Reply:
You are so right @henatan99, thank you for pointing this out. This error is now fixed.